package.json

```diff
@@ -1,37 +1,27 @@
 {
-  "private": true,
-  "scripts": {
```

Comment:
Let's not delete all these scripts and repurpose them? We still need these such as the publish and test scripts.

Reply:
I agree, but this part is not clear to me yet, I don't fully understand how git hooks works
If I just copy/paste the current scripts it will work ?